ros/src/styx/server.py

```diff
@@ -6,7 +6,7 @@
 import eventlet.wsgi
 import socketio
 import time
-from flask import Flask, render_template
+from flask import Flask  # , render_template
```

Comment:
Do we not need this import any more?

Reply:
Yea, it's not used so I was just cleaning it up.  I'll just delete it instead.